### PR TITLE
Fix findSimilarIncidents

### PR DESCRIPTION
**Pull request by erezh31**
1. add date-util embedded lib
2. fix date parsing issue

### CODE REVIEW

File: Scripts/script-FindSimilarIncidents.yml [hunk 1]
```diff
@@ -329,4 +342,4 @@ outputs:
 scripttarget: 0
 timeout: 300ns
 runonce: false
-releaseNotes: "Fix bugs: 1. handle exception for empty results, 2. support more that one incident keys"
\ No newline at end of file
+releaseNotes: "Fix date parsing bug"
```
Comment (meirwah) [inline Scripts/script-FindSimilarIncidents.yml]: I think you need to append to RN no?

Comment (aviadl) [reply Scripts/script-FindSimilarIncidents.yml]: In the same content release yes!

**Comment by erezh31** [comment]
@meirwah can u plz approve?

**Comment by anara123** [comment]
@erezh31 can be merged?

**Pull request merged**